### PR TITLE
Fix issue when release is RARred

### PR DESCRIPTION
**Pull request by fernandog**
and user has 'postpone if no subtitles'

As talked a couple of days ago in IRC

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by medariox** [comment]
I still see the problems we talked about. This can't be merged util they are fixed.

File: sickbeard/subtitles.py [hunk 1]
```diff
@@ -343,6 +343,25 @@ def subtitles_download_in_pp():  # pylint: disable=too-many-locals, too-many-bra
...
+                    if sickbeard.UNPACK and u'_UNPACK' not in root:
+                        result = processTV.ProcessResult()
+                        rar_content = processTV.unRAR(root, rar_files, True, result)
+                        # Only move if files are not in root folder of PP folder
+                        if rar_content and root != sickbeard.TV_DOWNLOAD_DIR:
+                            for extracted_file in rar_content:
+                                logger.log(u'Moving file to post-process folder: {}'.format(extracted_file), logger.DEBUG)
+                                moveFile(os.path.join(root, extracted_file), sickbeard.TV_DOWNLOAD_DIR)
+                            logger.log(u'Deleting folder: {}'.format(root), logger.DEBUG)
+                            processTV.delete_folder(root, False)
```
Comment (miigotu) [inline sickbeard/subtitles.py]: what if the user wants to keep seeding the rar? this will just blindly delete it.

**Comment by fernandog** [comment]
@medariox programs that create torrents don't allow multi-files without folder, so we won't have .rar files in the root fo PP - @duramato

File: sickbeard/subtitles.py (hunk 1 above)
Comment (fernandog) [inline sickbeard/subtitles.py]: @miigotu 
hum. do you see a way to only delete the .rar files without knowing them ? I didn't want to do another os.walk to list .r*

Comment (fernandog) [inline sickbeard/subtitles.py]: how this works in PP? afaik it deletes the .rar files isn't?
it deletes the .rar in processTV.py

Comment (miigotu) [inline sickbeard/subtitles.py]: in PP it doesnt delete the rars if process method is not 'move', and not in manual PP if 'delete_on' is not set.

``` python
if not(process_method == u"move" and result.result) or (proc_type == u"manual" and not delete_on):
    continue
```

and

``` python
if all([not sickbeard.NO_DELETE or proc_type == u"manual",
        process_method == u"move",
        ek(os.path.normpath, processPath) != ek(os.path.normpath, sickbeard.TV_DOWNLOAD_DIR)]):
```

**Comment by fernandog** [comment]
@miigotu new commit. please check

**Comment by duramato** [comment]
Yes you can only create a torrent of a single file like a .mkv or of a folder

**Comment by miigotu** [comment]
This would unrar the same rar every time the post processor runs, and does not avoid the problem in PP.

**Comment by fernandog** [comment]
fixed

File: sickbeard/processTV.py [hunk 2]
```diff
@@ -474,6 +474,10 @@ def already_postprocessed(dirName, videofile, force, result):  # pylint: disable
     """
     if force:
         return False
+        
+    if sickbeard.POSTPONE_IF_NO_SUBS and sickbeard.UNPACK:
+        # Return True because file was already unpacked in 'postpone if no subs' feature
+        return True
```
Comment (miigotu) [inline sickbeard/processTV.py]: and this will prevent every file from post processing later...

**Comment by fernandog** [comment]
@miigotu done. What do you think?

**Comment by medariox** [comment]
What about this? Is the processTV part okay?

**Pull request merged**